Core/SaveState.cpp

```diff
@@ -550,6 +550,27 @@ namespace SaveState
 		return false;
 	}
 
+	bool operator > (const tm &t1, const tm &t2) {
+		if (t1.tm_year > t2.tm_year) return true;
+		if (t1.tm_year < t2.tm_year) return false;
+		if (t1.tm_mon > t2.tm_mon) return true;
+		if (t1.tm_mon < t2.tm_mon) return false;
+		if (t1.tm_mday > t2.tm_mday) return true;
+		if (t1.tm_mday < t2.tm_mday) return false;
+		if (t1.tm_hour > t2.tm_hour) return true;
+		if (t1.tm_hour < t2.tm_hour) return false;
+		if (t1.tm_min > t2.tm_min) return true;
+		if (t1.tm_min < t2.tm_min) return false;
+		if (t1.tm_sec > t2.tm_sec) return true;
+		if (t1.tm_sec < t2.tm_sec) return false;
+		return false;
+	}
+
+	bool operator ! (const tm &t1) {
+		if (t1.tm_year | t1.tm_mon | t1.tm_mday | t1.tm_hour | t1.tm_min | t1.tm_sec) return false;
```

Comment:
Hmm, bitwise `|` seems odd here, although it might be fine.  Maybe better to use `||` for clarity?

-[Unknown]

Reply:
Bitwise works.  Was quite pleased with that solution.  I'd brushed up on them the week before and as I was overloading I was thinking low level.
But as the members of the `tm` struct are only `int` I can use `||` just fine.  And as you say that's more readable.